scripts/settings.js

```diff
@@ -469,7 +469,20 @@ async function setAnalytics(level, fSilent = false) {
  * Log out from the current wallet
  */
 export async function logOut() {
-    Database.removeIstance();
+    const fContinue = await confirmPopup({
+        title: 'Are you sure?',
+        html: `
+        <b>${tr(translation.netSwitchUnsavedWarningSubtitle, [
+            { network: cChainParams.current.name },
+        ])}</b>
+        <br>
+        This will delete all your data, including masternodes contacts and private keys!
+        <br>
+        <br>
+    `,
+    });
+    if (!fContinue) return;
+    Database.removeInstance();
```

Comment:
Nit: Should we `await` this? I don't see much harm happening, but I think making sure most DB interactions are async/awaited can save us some future race issues.